apps/prairielearn/python/prairielearn/html_utils.py

```diff
@@ -116,6 +132,16 @@ def _get_attrib(
     if len(args) > 1:
         raise ValueError("Only one additional argument is allowed")
 
+    # libxml2 sometimes changes the content of boolean attributes, so their
```

Comment:
Nit (IMO the previous wording made it seem like it was non-deterministic, but it's always done with the same set of attributes):

```suggestion
    # libxml2 omits the content of certain boolean attributes during serialization, so their
```

Consider rewrapping.